vars/piperExecuteBin.groovy

```diff
@@ -43,6 +43,7 @@ void call(Map parameters = [:], stepName, metadataFile, List credentialInfo, fai
                     sh "./piper ${stepName}${defaultConfigArgs}${customConfigArg}"
                 }
                 jenkinsUtils.handleStepResults(stepName, failOnMissingReports, failOnMissingLinks)
+                script.commonPipelineEnvironment.readFromDisk(script)
```

Comment:
Just curious, why does it need to happen in the closure? For symmetry I would expect it on the same indentation level as line 26.

Reply:
I added it there in order to avoid potential stashing problems on the way back to the Jenkins.
We may move the `writeToDisk()` also into the closure to make sure that the environment is always available to the runtime ...

Reply:
Sounds good.

Reply:
I guess we need to leave the `writeToDisk()` where it is since it might get relevant for the first config call.

If we see at some point that this requires some changes, let's do this in a separate PR.

Reply:
@stippi2 what do you think, should we continue like it is?

Reply:
Sure, absolutely.